pkg/worker/integrate/manager.go

```diff
@@ -0,0 +1,77 @@
+package integrate
+
+import (
+	"fmt"
+
+	"github.com/caicloud/cyclone/pkg/api"
+)
+
+// ITGProvider is an interface for ingetration.
+type ITGProvider interface {
```

Comment:
`ITG` is a little confused, use full word?

Reply:
Full word is a little long.